lib/Room.php

```diff
@@ -701,9 +703,15 @@ public function changeInCall($sessionId, $active) {
 		$query->execute();
 
 		if ($active) {
-			$this->dispatcher->dispatch(self::class . '::postSessionJoinCall', new GenericEvent($this, [
-				'sessionId' => $sessionId,
-			]));
+			if ($hadSessionsInCall) {
+				$this->dispatcher->dispatch(self::class . '::postSessionJoinCall', new GenericEvent($this, [
+					'sessionId' => $sessionId,
+				]));
+			} else {
+				$this->dispatcher->dispatch(self::class . '::postSessionStartCall', new GenericEvent($this, [
```

Comment:
This fucks up the external signaling and others.
I would just to the `hasSessionsInCall` check in the listener of the current event.